gemini/src/main/java/com/techempower/gemini/HttpRequest.java

```diff
@@ -1,3 +1,29 @@
+/*******************************************************************************

```

Comment:
Unrelated to this PR, at some point can we go through the entire project and change all of the copyright comment blocks to start/end with `/* */` rather than `/****... */`? My IDE keeps giving me warnings about this being a dangling javadoc comment, which it technically is (it's attached to nothing), and it'd be nice to not see warnings in every single class. I could just turn off that warning, but it's a helpful warning outside this specific use case.

Reply:
Yes, we can; we need to update the copyright year anyway.

Reply:
For what it's worth, this is the recommended approach https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html#3441